Modules/Invoices/Controllers/AjaxController.php

```
@@ -290,13 +292,13 @@ public function modalCopyInvoice(Request $request) {
      * If the "einvoicing" setting is enabled, may adjust the legacy_calculation configuration based on input.
      */
     public function copyInvoice(Request $request) {
-        if ($this->invoicesService->runValidation()) {
+        if ($this->invoicesService->runValidation(null, $request)) {
             // Automatic calculation mode
             if (get_setting('einvoicing')) {
                 // Shift to false (by default). Need true? See Dev Note on ipconfig example
                 config(['legacy_calculation' => ! empty($request->post('legacy_calculation'])));
```

PHPStan (2.1.31)

Syntax error, unexpected ']', expecting ')' on line 299


(phpstan.parse)